### DISCUSSION TITLE
希望能部署到子目录

### DISCUSSION THREAD

**Discussion by ghost** [walinejs/waline#130] [Q&A] [answered]
比如 `https://waline.example.com/waline`。

现在的情况就是反代到子目录以后，管理页面呈现空白。

我用的平台是 cloudbase。

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/tcb-starter cloudbase 用这个模板吧，默认部署到 `/waline` 下，后续官方文档那边会修改下。

**Reply by ghost** [reply]
就是用的这个

**Reply by ghost** [reply]
我的锅，浏览器缓存刷新了就好了，没事了，没事了

**Reply by lizheming** [reply]
好的，没事，感谢反馈。
